core/modules/simpletest/tests/form_test.module

```diff
@@ -771,6 +771,13 @@ function _form_test_tableselect_js_select_form($form, $form_state, $action) {
   return _form_test_tableselect_form_builder($form, $form_state, $options);
 }
 
+/**
+* AJAX callback that returns the form element.
+*/
+function _form_test_tableselect_ajax_callback($form, &$form_state) {
+  return $form['tableselect'];
+}
+
```

Comment:
Simply moving this slightly up, to be with the rest of the `_form_test_tableselect_*()` functions.